real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/GpsPanel.java

```diff
@@ -47,15 +25,27 @@
 	private double theta = 0;
 	private MapPolygonImpl directionLine;
 	
-	private JMapViewerTree mapTree;
-	
-	private double mapViewerLat = 40.440138;
-	private double mapViewerLon = -79.945306;
-	private String mapCacheFolderDiskPath = "images/cachedCourseMap";
-	
-	private double mapDragX = -1;
-	private double mapDragY = -1;
-	
+	private Map map;
+	private MapMarkerDot destinationPoint;
+
+	/**
+	 * @return the destinationPoint
+	 */
+	public synchronized MapMarkerDot getDestinationPoint() {
+		return destinationPoint;
+	}
+
+
+	/**
+	 * @param destinationPoint the destinationPoint to set
+	 */
+	public synchronized void setDestinationPoint(MapMarkerDot destinationPoint) {
+		this.destinationPoint.setLat(destinationPoint.getLat());
+		this.destinationPoint.setLon(destinationPoint.getLon());
+
+	}
+
+
 	private GPSPoseMessage lastpose;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2362883620/issues/source?bid=2938665&fileBranchId=3216826#l49)